packages/math-input/src/components/input/math-wrapper.ts

```diff
@@ -240,13 +141,6 @@ class MathWrapper {
 
     /**
      * Place the cursor beside the node located at the given coordinates.
-     *
-     * @param {number} x - the x coordinate in the viewport
-     * @param {number} y - the y coordinate in the viewport
-     * @param {Node} hitNode - the node next to which the cursor should be
-     *                         placed; if provided, the coordinates will be used
-     *                         to determine on which side of the node the cursor
-     *                         should be placed
```

Comment:
Why were these comments deleted?  They seem useful. :)

Reply:
It felt weird mixing JSDoc and TypeScript, but I don't know why I felt that. Added back. 👍